stacklight_tests/tests/test_smoke.py

```diff
@@ -1,7 +1,105 @@
+            3. Check that InfluxDB is online and can serve requests
+
+        Duration 1m
+        """
+        service = "influxdb"
+        self.check_service_installed(service)
+        self.check_service_running(service)
+        measurements, env_name = self.influxdb_api.check_influxdb_online()
+        assert measurements and env_name
+
+    def test_grafana_installed(self):
+        """Smoke test that checks basic features of Grafana.
+
+        Scenario:
+            1. Check Grafana package is installed
+            2. Check Grafana is up and running
+            3. Check that user can login into and HTTP API is working
+            4. Check that access prohibited for non-authorized user
+
+        Duration 1m
+        """
+        self.check_service_installed("grafana")
+        self.check_service_running("grafana-server")
+        self.grafana_api.check_grafana_online()
+
+    def test_nagios_installed(self):
+        """Smoke test that checks basic features of Nagios.
+
+        Scenario:
+            1. Check that hosts page is available
+            2. Check that services page is available
+            3. Check that access prohibited for non-authorized user
+
+        Duration 1m
+        """
+        hosts = self.nagios_api.get_all_nodes_statuses()
+        services = self.nagios_api.get_all_services_statuses()
+        assert hosts and services
+        # Negative testing
+        origin_password = self.nagios_api.password
+
+        def set_origin_password():
+            self.nagios_api.password = origin_password
+            self.nagios_api.nagios_url = self.nagios_api.format_url()
+        self.destructive_actions.append(set_origin_password)
+        self.nagios_api.password = "rogue"
+        self.nagios_api.nagios_url = self.nagios_api.format_url()
+        for page in self.nagios_api.pages.keys():
+            self.nagios_api.get_page(page, expected_codes=(401,))
+        set_origin_password()
+        self.destructive_actions = []
+
+    def test_elasticsearch_installed(self):
+        """Smoke test that checks basic features of Elasticsearch.
+
+        Scenario:
+            1. Check Elasticsearch package is installed
+            2. Check Elasticsearch is up and running
+            3. Check that elasticsearch is online
+            4. Check logs queries
+            5. Check notification queries
+
+        Duration 1m
+        """
+        service = "elasticsearch"
+        self.check_service_installed(service)
+        self.check_service_running(service)
+        log_result = self.elasticsearch_api.query_elasticsearch(size=10)
+        log_failed_shards = log_result["_shards"]["failed"]
+        log_hits = log_result["hits"]
+        notify_result = self.elasticsearch_api.query_elasticsearch(size=10)
+        notification_failed_shards = notify_result["_shards"]["failed"]
+        notification_hits = notify_result["hits"]
+        assert ((not log_failed_shards) and log_hits and
```

Comment:
I think it's antipattern for testing process, because it wouldn't show full status of checking features, but it would show only one on which it will fail.